common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -98,6 +107,37 @@
 
     private static final String TAG = DevicePopManager.class.getSimpleName();
 
+    /**
+     * Signing algorithms supported by our underlying keystore. Not all algs available at all device
+     * levels.
+     */
+    public static class SigningAlgorithms {
```

Comment:
Could/should this be an enum?

Reply:
Will leave as a string, these are as-is from the documentation and are [supplied to the keystore as Strings ](https://developer.android.com/training/articles/keystore#WorkingWithKeyStoreEntries)-- that's Android's API. Having these as enums means conversion between types...(enum -> string). Since this isn't public API, I'm inclined to leave as-is

See:
https://developer.android.com/training/articles/keystore#SupportedSignatures

Reply:
Eh.  trading a call to .name() on the enum for compile-time argument checking is fairly trivial in my experience.

Reply:
Since this is proving to be popular feedback, I'll take this change after all

Reply:
ef45a531f4458dca339b34f159080bfbd1b39bf1

Reply:
Minor note: using `.toString()` instead of `.name()` because the enum cannot contain `/` chars in its declaration, as is required for RSA w/ PSS